compiler/compiler/src/main/java/org/robovm/compiler/target/AbstractTarget.java

```diff
@@ -539,6 +555,11 @@ protected void stripArchive(Path path, File output) throws IOException {
                             // Don't include anything under META-INF/robovm/
                             continue;
                         }
```

Comment:
I think we should add those exceptions to the robovm.xml and as default in the template. So we don't need to maintain two branches for the same functionality.